<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="busServiceList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>201</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="getBusRoute">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>951</width>
     <height>711</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QLineEdit" name="fromTextField">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="getRouteButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>60</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>From:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>60</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>To:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="toTextField">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>View Bus Routes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Route</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
